### DISCUSSION TITLE
Are we supposed to use the Postgres timezone columns different than with app timezone value?

### DISCUSSION THREAD

**Discussion by ProteanCode** [laravel/framework#39002] [Q&A]
No matter how hard I try I cannot save a custom timezone when saving a model

The app configuration is running under UTC timezone, but i want to save a column with Europe/Warsaw timezone and that does not work.

```
$value = Carbon::now()->setTimezone('Europe/Warsaw');
$model->foo = $value;
$model->save();
```

No matter what the dates in database are saved as `+00` which means `UTC`

**Comment by tpetry** [comment]
The timezone information is stored with the value in PostgreSQL. But it's not enforcing the timezone you store, it's internally stored in UTC. The time zone for retrieving the value is dependent on your session, you can change it with `SET TIME ZONE`.
